catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/transformers/MiroTransformableTransformer.scala

```diff
@@ -69,6 +69,7 @@ class MiroTransformableTransformer
           language = None,
           dimensions = None,
           items = getItems(miroData, miroTransformable.sourceId),
+          itemsV1 = getItemsV1(miroData, miroTransformable.sourceId),
```

Comment:
keyword arguments please? :)

Reply:
I don't think they add anything here, to be honest, other than making the line longer